weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallOverview.tsx

```diff
@@ -18,13 +18,13 @@ export const Overview = styled.div`
 `;
 Overview.displayName = 'S.Overview';
 
-export const CallName = styled.div<{$isEditing?: boolean}>`
+export const CallName = styled.div`
   font-family: Source Sans Pro;
   font-size: 16px;
   font-weight: 600;
   text-align: left;
   word-break: break-all;
-  width: ${props => (props.$isEditing ? '100%' : 'max-content%')};
+  width: 100%;
```

Comment:
@m-rgba - I think the `max-content` won't work inside the flex container regardless but the trailing `%` on `max-content` makes that value invalid and ignored anyway.